File: c/128/70.c
int climbStairs(int n){
  int f1 = 1, f2 = 1, tmp;
  for (int i = 1; i < n; i++) {
    tmp = f2;
    f2 += f1;
    f1 = tmp;
  }
  return f2;
}

File: py3/128/70.py
class Solution:
    def climbStairs(self, n: int) -> int:
        if n < 3: return n
        f1, f2 = 1, 2
        while n > 2:
            tmp = f2
            f2 += f1
            f1 = tmp
            n -= 1
        return f2